docs/templates/toc.yml

```diff
@@ -72,6 +72,8 @@
       name: Series
     - name: Window
       uid: bigframes.core.window.Window
+    - name: Supported pandas APIs
+      href: supported_pandas_apis.html
```

Comment:
@ashleyxuu Is this the right way to do this? I can't figure out how to get the HTML file to stage.

Reply:
```
- href: supported_pandas_apis.html
      name: Supported pandas APIs

Reply:
Done.